peter_lin/midterm/lib/catlady.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'client'
+
+class CatLady < Client
+
+  def initialize (name,age)
+    @name = name
+    @age = age
+    @pets = []
+  end
+
+  def take_pet(animal)
+    if animal.species == "cat"
```

Comment:
great job with this. i can tell you really soaked up object oriented programming concepts.